packages/xo-server-cloud/src/index.js

```diff
@@ -20,8 +20,10 @@ class XoServerCloud {
   }
 
   async load() {
-    const getResourceCatalog = () => this._getCatalog()
-    getResourceCatalog.description = 'Get the list of all available resources'
+    const getResourceCatalog = ({ filters } = {}) =>
+      this._getCatalog({ filters })
+    getResourceCatalog.description =
+      "Get the list of user's available resources"
     getResourceCatalog.permission = 'admin'
```

Comment:
You need to declare supported params, example: https://github.com/vatesfr/xen-orchestra/blob/47819ea95620880579111ca9d25bbfda8ec82acc/packages/xo-server/src/api/xo.js#L55-L59